docs/source/linchpincli_linchpin_destroy.rst

```diff
@@ -0,0 +1,35 @@
+linchpin destroy
+================
+
+Usage: ``linchpin destroy [OPTIONS] TARGET``
+
+Destroys nodes from the given target(s) in the given PinFile.
+
+**Arguments**
+
+``TARGET ...``
+	Destroy ONLY the listed target(s). If omitted, ALL targets are destroyed.
+
+**Options**
+
+``-p, --pinfile TEXT``
+	Use a name for the PinFile different from the configuration.
```

Comment:
The PinFile is not found in the configuration, but rather in the workspace directory. You can also set the $WORKSPACE env variable. If the user prefers to pass in the pinfile on the cli, it would use this instead of the one in the workspace.